provider/k8s.go

```diff
@@ -99,12 +99,15 @@ type K8sOfflineStore struct {
 	BaseProvider
 }
 
-func (store *K8sOfflineStore) AsOfflineStore() (OfflineStore, error) {
-	return store, nil
+func (k8s *K8sOfflineStore) AsOfflineStore() (OfflineStore, error) {
+	return k8s, nil
 }
 
 func (k8s *K8sOfflineStore) Close() error {
-	k8s.store.Close()
+	err := k8s.store.Close()
+	if err != nil {
```

Comment:
would it be fine if we just return `err` unless we want to add an error message?

Reply:
yeah will simplify